profile_library/signify/915005997901/model.json

```
@@ -0,0 +1,15 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2025-12-11T09:34:01.728982",
+  "device_type": "light",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "shelly 1PM Gen4",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.19.2:docker"
+  },
+  "name": "Phillips Hue Xamento",
```

```suggestion
  "name": "Hue Xamento",
```

Typo in Philips, but I would suggest to keep it in line with the other models and just call it Hue